### ISSUE TITLE
Missing year and and owner in license copyright

### ISSUE THREAD

**Issue by radimsuckr** [dagger/dagger#2156]
### What is the issue?

Hi,

I found a minor issue in the license file. On line 190 there's `Copyright [yyyy] [name of copyright owner]` which seems like it's missing a year and copyright owner. :slightly_smiling_face:

**Comment by thaJeztah** [comment]
No, that's intentional; that part in the LICENSE file is a template that you can use to apply the license to the code; to apply the Apache license to a project, either

- add a NOTICE file describing the copyright owner
- add a license headers to each source file (using that section from the LICENSE as an example)
- or both

Also see, e.g. https://github.com/kubernetes/kubernetes/commit/d30db1f9a915aa95402e1190461469a1889d92be

**Comment by thaJeztah** [comment]
That said; it looks like currently the repository is missing both, which could mean it's technically "not licensed" (but looking from my phone so possibly looking wrong) so probably needs something like https://github.com/docker/cli-docs-tool/pull/2

/cc @shykes

**Comment by shykes** [comment]
Hi @thaJeztah ! Good to see you here :)

Just to clarify, you're saying that we should add either:

1. A `NOTICE` file, or
2. A license header to each source file?

**Comment by radimsuckr** [comment]
Oh, sorry for misreport then. :sweat_smile: At least I learned something new!

**Comment by shykes** [comment]
Thanks to this issue, we caught the missing NOTICE file, and fixed that. Thank you @radimsuckr and @thaJeztah !
